python/ray/tune/automlboard/README.md

```diff
@@ -0,0 +1,6 @@
+## About AutoMLBoard
+
+AutoMLBoard can be used as a monitor who collect information about ray tune jobs and
+show the trial information to the frontend.
+
+Please refer to `run.py -h` for more help
```

Comment:
It'd be good to provide more instructions here, e.g., what are the installation dependencies? Does the user need to do `pip install django`?

Reply:
yeah, ``pip install jango`` is needed, I'll add some instructions here.